### ISSUE TITLE
MS5637_ADDRESS 

### ISSUE THREAD

**Issue by funwey** [kriswiner/MPU-9250#317]
Hi Kris;

I am looking at the MPU9250_MS5637_AHRS_t3 code:
#if ADO
#define MPU9250_ADDRESS 0x69  // Device address when ADO = 1
#define AK8963_ADDRESS 0x0C   //  Address of magnetometer
#define MS5637_ADDRESS 0x76   // Address of altimeter
#else
#define MPU9250_ADDRESS 0x68  // Device address when ADO = 0
#define AK8963_ADDRESS 0x0C   //  Address of magnetometer
#define MS5637_ADDRESS 0x76   // Address of altimeter
#endif  

I guess the MS5637_ADDRESS should be 0x77 for ADO=1 , while AK8963_ADDRESS  remain the same ?

Rgds

Fun Wey

**Comment by funwey** [comment]
I presume that the MS5637 is Borsch's BMP 280 barometer?

**Comment by kriswiner** [comment]
If you have one of my MPU9250+ MS5637 boards then the MS5637 address is
always 0x76. AAK8963C address is always 0x0C.

**Comment by kriswiner** [comment]
No, the MS5637 is Measurement Specialty's pressure sensor, not the BMP280.
Different animal.
